readthedocs/doc_builder/python_environments.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, version, build_env, config=None):
         if config:
             self.config = config
         else:
-            self.config = ConfigWrapper(version=version, yaml_config={})
+            self.config = load_yaml_config(version)
```

Comment:
This was the only part of the code where we were creating the `ConigWrapper` object directly. I have changed this, but I have to change this https://github.com/rtfd/readthedocs.org/pull/4298/files#diff-8b90ee75bb4ed7a0341eac84832b7ea7R166

Reply:
I think we can remove the else here, `config` is always passed in our code